src/app/components/Sandbox.js

```diff
@@ -452,9 +462,14 @@ const SandboxComponent = ({ admin }) => {
                 </div>
               }
               { ((listItemMedia && !listItemShared) || (listItemShared && listItemCluster)) &&
+              <>
+                <div className={styles.thumbail}>
+                  <ItemThumbnail picture={mediaThumbnail.media?.picture} maskContent={mediaThumbnail.show_warning_cover} type={mediaThumbnail.media?.type} url={mediaThumbnail.media?.url} />
+                </div>
```

Comment:
- [ ] I understand about the refactoring needed for the content warning overlay for super admin, thanks for the explanation, makes sense to open a ticket for that. For now lets remove the content warning version from the sandbox and just leave the "regular" one, so its more realistic